docs/release-notes/command-startup.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  Commands: Commands now support automatically running a ``startup-hook.sh`` when one is present in
+   the commands context directory.
```

Comment:
- Commands: Commands now support automatically executing a ``startup-hook.sh`` script if it is present in the command's context directory.